packages/css/src/paragraph/paragraph.scss

```diff
@@ -5,28 +5,45 @@
 
 @import "../../utils/calculate-fluid-style";
 
+@import "../../node_modules/@amsterdam/design-system-tokens/dist/variables";
+
 @mixin reset {
   box-sizing: border-box;
   -webkit-text-size-adjust: 100%;
 }
 
 .amsterdam-paragraph {
-  color: var(--amsterdam-color-primary-black);
-  font-family: var(--amsterdam-typography-font-family);
-  font-size: calculate-fluid-style(18, 24);
-  font-weight: var(--amsterdam-typography-font-weight-normal);
-  line-height: calculate-fluid-style(29, 38);
+  color: var(--amsterdam-paragraph-color);
+  font-family: var(--amsterdam-paragraph-font-family);
+
+  /*
+    We don't use the amsterdam-paragraph-font-size variable, because we generate a somewhat
+    complex css rule with a Sass function. The variable is added here so people can
+    overwrite that css rule with their own value.
+  */
+  font-size: var(
+    --amsterdam-paragraph-font-size,
+    calculate-fluid-style($amsterdam-typography-font-size-18, $amsterdam-typography-font-size-24)
```

Comment:
A couple of thoughts on this function, although it’s technically out of scope of this PR.

First, I’d like to improve its name. It returns the CSS clamping function with a fluid preferred value. How about `clamped-fluid-value`?

Reading up a bit (from [here](https://css-tricks.com/snippets/css/fluid-typography/) to [there](https://css-tricks.com/simplified-fluid-typography/)) it seems that we should be able to rewrite this preferred value into the form of `(a)rem + (b)vw`. Let’s try that.

And if we get the function to return a static value in case both parameters are equal, we might even be able to get rid of the unused variable, although that is a fancy trick.

Reply:
Additionally, should we introduce component variables like `$amsterdam-paragraph-font-size-min` and `$amsterdam-paragraph-font-size-max` instead of directly using the typography tokens?

Reply:
Regarding the name: `clamped-fluid-value` isn't very intuitive imo. Clamping is **how** you do it, not **what** you're doing. Maybe something like `generate-fluid-value`? Of even simpler: `fluid-value`?

Simplifying the function sounds like a good idea, but maybe we should wait until we know what the function should actually do. The design is still unclear on this. 

I don't quite get your third remark. If both parameters are equal, you shouldn't be using this function, right?

I thought about adding a font min and max to the design tokens, but this would sort of force external users to use fluid typography, right? By using a single design token, you can choose whether you want to use fluid typo

Reply:
Sure, let’s make a separate ticket and branch for this. And `fluid-value` is a great name.